src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -118,6 +118,11 @@ public void setAddressBook(ReadOnlyAddressBook newData) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
```

Comment:
not a major problem for now but you might want to try adding new import command test cases e.g. import empty file, wrong paths etc :))